am-ui/webcomponents/create-ad-webcomponent/package.json

```diff
@@ -11,30 +11,66 @@
     "preview": "vite preview",
     "openapi-gen": "openapi-generator-cli generate -g javascript -i ../../../api.yaml -o ./src/api",
     "package": "node package-script.js",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore --fix src"
+    "test:unit": "vue-cli-service test:unit",
```

Comment:
Why are you using vue-cli-service for test and lint? Calling eslint for the lint command as before should work and calling `jest` for the test command (no need to specify `:unit` either, since it'll be the only kind of tests we'll be doing here) works too.
`vue-cli-service` (line 37) shouldn't be installed at all, since we're using `vite` which replaces it.

Reply:
Very true, fixed